pkg/webhooks/utils/policy_context_builder.go

```diff
@@ -133,6 +69,11 @@ func (b *policyContextBuilder) Build(request *admissionv1.AdmissionRequest, poli
 	if err := ctx.AddImageInfos(&newResource); err != nil {
 		return nil, errors.Wrap(err, "failed to add image information to the policy rule context")
 	}
+	userRequestInfo.Roles, userRequestInfo.ClusterRoles, err = userinfo.GetRoleRef(b.rbLister, b.crbLister, request, b.configuration)
```

Comment:
better to call this before `newVariablesContext`

Reply:
yes

Reply:
@vyankyGH not fixed right ?

Reply:
done